decorator.rb

```diff
@@ -0,0 +1,26 @@
+require './nameable'
+
+class Decorator < Nameable
+  attr_accessor :nameable
+
+  def initialize(nameable)
+    @nameable = nameable
+    super()
+  end
```

Comment:
- [ ] May I know why you called the `super()` method in `initialize`?

Reply:
I called `super()` because when it is absent, Rubocop flags it as an offence (as can be seen in the screenshot below).

![super](https://user-images.githubusercontent.com/88531553/163117508-1b2267ce-5539-4977-817e-591422ff7b26.png)

Reply:
Ok, I understand